src/Stryker.Core/Stryker.Core/InjectedHelpers/Coverage/StaticContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stryker
+{
+    public sealed class StaticContext: IDisposable
+    {
+        [ThreadStatic] private static int depth;
+
+        public StaticContext()
+        {
+            depth++;
+        }
+
+        public void Dispose()
+        {
+            depth--;
+        }
+
+        public static bool InStatic()
```

Comment:
This is a deliberate choice. This expresses if the code being executed has been called by some static initialiser. Even if the method/accessor itself is not static.
Not sure _InStatic_ is the best naming, but I think _IsStatic_ would be misleading.

Reply:
Perhaps it makes more sense if the class is simply called Context, CoverageContext, MutantContext or something like that. Then it would be Context.InStatic or Context.Static, Context.WhateverOtherContext we need later on.

Reply:
Renamed the class __StaticContext__ to __MutantContext__ for clarity.